ts/torch_handler/base_handler.py

```diff
@@ -141,7 +151,10 @@ def preprocess(self, data):
         Returns:
             tensor: Returns the tensor data of the input
         """
-        return torch.as_tensor(data, device=self.device)
+        t = torch.as_tensor(data, device=self.device)
+        if ipex_enabled:
+            t = t.to(memory_format=torch.channels_last)
+        return t
```

Comment:
@HamidShojanazeri currently this only apply to vision models, because in PyTorch only 4D tensor can be changed to NHWC.